shopping-server/package.json

```diff
@@ -30,7 +30,7 @@
     "jsonwebtoken": "^8.4.0",
     "lodash": "^4.17.11",
     "mongodb": "^3.1.10",
-    "mongoose": "^5.3.15",
+    "mongoose": "^5.9.12",
```

Comment:
There is already another PR out with this change. Thank you.

Reply:
Okay